pennylane/devices/default_qubit_autograd.py

```diff
@@ -84,7 +84,6 @@ class DefaultQubitAutograd(DefaultQubit):
     _reduce_sum = staticmethod(lambda array, axes: np.sum(array, axis=tuple(axes)))
     _reshape = staticmethod(np.reshape)
     _flatten = staticmethod(lambda array: array.flatten())
-    _gather = staticmethod(lambda array, indices: array[indices])
```

Comment:
This redefinition is actually unnecessary, because it does not use any package. Do we want to keep it nonetheless?